docs/stsynphot/appendixb_nonhst.rst

```diff
@@ -285,34 +285,32 @@ The Stromgren *uvby* throughputs are taken from
     >>> bp = stsyn.band('stromgren,y')  # doctest: +SKIP
 
 
-.. _stsynphot-nonhst-wfirst:
+.. _stsynphot-nonhst-roman:
```

Comment:
Hmm... this will break any intersphinx linking in other repos. Do we really need to change this? It is not really visible to the user.

Reply:
I think we should change it now rather than later. The mission has been pretty set on removing all the old references to WFIRST in favor of Roman, and I'd prefer to get things changed over earlier while it impacts few if any people rather than later when it could impact more.